esphome/components/rtsp_server/rtsp_server.cpp

```diff
@@ -0,0 +1,131 @@
+#include "rtsp_server.h"
+#include "esphome/core/log.h"
+#include "esphome/core/application.h"
+#include <StreamString.h>
+#include "esphome/core/util.h"
+
+
+namespace esphome {
+namespace rtsp_server {
+
+static const char *TAG = "rtsp_server";
+
+void RTSPServer::setup() {
+
+  if ( this->camera__ == nullptr) {
+     ESP_LOGCONFIG(TAG, "Error: Unable to obtain reference to camera");
+  }
+
+  camera_config_t cameraConfig =  this->camera__->get_camera_config();
+
+  this->dim = this->parseCameraDimensions(cameraConfig);
+
+  ESP_LOGCONFIG(TAG, "Beginning to set up RTSP server listener");
```

Comment:
```suggestion
  ESP_LOGD(TAG, "Beginning to set up RTSP server listener");
```

config log level is only for things written in `dump_config`

Reply:
> config log level is only for things written in dump_config

We might need to clean up the codebase then =)